src/components/molecules/ReplyItem/ReplyItem.tsx

```diff
@@ -0,0 +1,185 @@
+import { useParseJwt } from '@/hooks/common/useParseJwt';
+import { useMemberQuery } from '@/hooks/api/member/useMemberQuery';
+import { formatDateFromISOWithTime } from '@/utils/formatData';
+import { useCommentActions } from '@/hooks/comment/useCommentActions';
+import MenuTap, { MenuItem } from '@/components/atoms/MenuTap/MenuTap';
+import ToastModal from '@/components/atoms/ToastModal/ToastModal';
+import LikeButton from '@/components/atoms/LikeButton/LikeButton';
+import CategoryBarChip from '@/components/atoms/CategoryBarChip/CategoryBarChip';
+import TextArea from '@/components/molecules/TextArea/TextArea';
+import TextModal from '@/components/molecules/TextModal/TextModal';
+import ReportModal from '@/components/molecules/ReportModal/ReportModal';
+import useToastModal from '@/hooks/modal/useToastModal';
+import * as S from './ReplyItem.style';
+
+export interface ReplyItemProps {
+  reply: Comment;
+  selectedPage: number;
+  talkPickWriter: string;
+  parentId: number;
+}
+
+const ReplyItem = ({
+  reply,
+  selectedPage,
+  talkPickWriter,
+  parentId,
+}: ReplyItemProps) => {
+  const accessToken = useNewSelector(selectAccessToken);
+  const { member } = useMemberQuery(useParseJwt(accessToken).memberId);
+
+  const isMyReply: boolean = reply?.nickname === member?.nickname;
+  const isTalkPickWriter: boolean = reply?.nickname === talkPickWriter;
+
+  const replyRef = useRef<HTMLDivElement>(null);
+  const { isVisible, modalText, showToastModal } = useToastModal();
+
+  const [reportModalOpen, setReportModalOpen] = useState<boolean>(false);
+  const [reportTextModalOpen, setReportTextModalOpen] =
+    useState<boolean>(false);
+  const [deleteTextModalOpen, setDeleteTextModalOpen] =
+    useState<boolean>(false);
+
+  const [editReplyClicked, setEditReplyClicked] = useState<boolean>(false);
+  const [editReplyText, setEditReplyText] = useState<string>(reply.content);
+
+  const { handleEditSubmit, handleDelete, handleLikeToggle, handleReport } =
+    useCommentActions(
+      reply,
+      editReplyText,
+      selectedPage,
+      setEditReplyClicked,
+      showToastModal,
+      parentId,
+    );
+
+  useEffect(() => {
+    setEditReplyText(reply.content);
+  }, [reply.content]);
+
+  useEffect(() => {
+    const handleClick = (e: MouseEvent) => {
+      if (replyRef.current && !replyRef.current.contains(e.target as Node)) {
+        setEditReplyClicked(false);
+      }
+    };
+    window.addEventListener('mousedown', handleClick);
+    return () => window.removeEventListener('mousedown', handleClick);
+  }, [replyRef]);
+
+  const handleDeleteReplyButton = () => {
+    setDeleteTextModalOpen(false);
+    handleDelete();
+  };
+
+  const myReply: MenuItem[] = [
+    {
+      label: '수정',
+      onClick: () => {
+        setEditReplyClicked(true);
+      },
+    },
+    {
+      label: '삭제',
+      onClick: () => {
+        setDeleteTextModalOpen(true);
+      },
+    },
+  ];
+
+  const otherReply: MenuItem[] = [

```

Comment:
```suggestion
  const reportReply: MenuItem[] = [
```
또는 조금 더 직관적인 명칭을 사용해도 괜찮을 거 같아요!

Reply:
[1a9682b](https://github.com/CHZZK-Study/Balance-Talk-Frontend/pull/202/commits/1a9682b73f98e5abf138e5aa9128add1507bd209) 에서 함께 수정했습니당